projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -792,6 +799,10 @@ private boolean answerType(
 
    private boolean cat(String[] words)
          throws IOException, FileNotFoundException {
+      if (words.length == 1) {
+         _logger.errorf(words[0] + " requires additional arguments\n");
```

Comment:
From the delAnalysis function:
      if (options.size() != 0 || parameters.size() != 1) {
         _logger.errorf("Invalid arguments: %s %s\n", options.toString(),
               parameters.toString());
         printUsage(Command.DEL_ANALYSIS);
         return false;
      }

Basically, an explanatory message of what was wrong, followed by usage of the command.